src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -37,28 +41,47 @@ class EmailNotificationForm extends Component {
         email: this.state.email,
         language: this.state.language,
       }
-      // this.props.postNewEmailSubscriber(newEmailSubscriber);
+
+      postNewEmailSubscriber(newEmailSubscriber)
+        // .then(response => {
+        //   if (response.status !== 200) {
+        //     throw new Error (`${response.status} : Sorry, cannot fetch the data.`)
+        //   }
+        //   if (!response.ok) {
+        //     throw new Error ('Something has gone wrong, please try again.')
+        //   }
+        //   return response.json()
+        // })
+        // .then(message => {
+        //    this.setState({ isSubmitting: true });
+        //    this.setState({ successMessage: message })''
+        // })
+        // .catch(error => {
+        //   this.setState({ isSubmitting: false });
+        //   this.setState({ error: error.message });
+        // })
+
```

Comment:
What do you think about adding this postNewEmailSubscriber method to the apiCalls.js file without hooks, importing it to App.js and using it there inside of a postNewEmailSubscriberToDB method and passing it down as props to the form component (i.e. without the endpoints.js, hooks.useFetch and, hooks.usePost files)?